src/three/plugins/ImplicitTilingPlugin.js

```diff
@@ -0,0 +1,63 @@
+import {SUBTREELoader} from "./SUBTREELoader.js";
+
+export class ImplicitTilingPlugin {
+
+	constructor() {
+
+		this.name = 'IMPLICIT_TILING_PLUGIN';
+
+	}
+
+	init(tiles) {
+
+		this.tiles = tiles;
+
+	}
+
+	preprocessNode(tile, uri, parentTile) {
+
+		if (tile.implicitTiling) {	//Check if the tile has a declaration of an implicitTiling
+
+			tile.__hasUnrenderableContent = true;
+			tile.__hasRenderableContent = false;
+
+			// Store the infos from the tileset
+			tile.__subtreeDivider =  tile.implicitTiling.subdivisionScheme === "QUADTREE" ? 4 : 8;
+			tile.__subtreeUri = tile.implicitTiling.subtrees.uri;
+
+			// Original content uri
+			tile.__contentUri = uri ?? tile.content?.uri;
+
+			// Declare some properties
+			tile.__subtreeIdx = 0;	// Idx of the tile in its subtree
+
+			// Coords of the tile
+			tile.__x = 0;
+			tile.__y = 0;
+			tile.__z = 0;
+			tile.__level = tile.__depth;
```

Comment:
Since this is _always_ the implicit root of sub tree should the level no always start off at 0? From the spec:

> level is `0` for the implicit root tile. This tile’s children are at level 1, and so on.